source/_integrations/utility_meter.markdown

```diff
@@ -136,14 +136,21 @@ utility_meter:
 ```
 
 Assuming your energy provider tariffs are time based according to:
-
 - *peak*: from 9h00 to 21h00
 - *offpeak*: from 21h00 to 9h00 next day
 
-a time based automation can be used:
+As energy companies (certainly in the UK, possibly elsewhere) time from midnight, this equates to:
+- *offpeak*: 0h00 to 9h00
+- *peak*: from 9h00 to 21h00
+- *offpeak*: from 21h00 to 0h00
+
+You need TWO automations. 
+- a time based automation can be used to change the tariffs. This will switch to "next" tariff at the time point.
+- on restart automation to set the correct tariff in the first place, so you are at the right tariff on startup at any time of day.
 
 ```yaml
 automation:
+- alias: 'Trigger changes to tariff'
```

Comment:
Why do we need 2 automations for this? That feels weird, as you can trigger on start, the set times and decide in the action (using a chooser) on what to execute?

Reply:
@frenck 
I have a lot of changes in tariff (see my other comment) that I thought it easier to set the right tariff initially, then flip between the two tariffs separately.

Reply:
In both cases, a state is set on an event (e.g., change of time or startup of Home Assistant). The event might differ, but determining the tariff is the same. Thus one automation can be used.

Reply:
I’m new to HA, so I’m trying to understand the options. If there is a better way of doing this, can you print me at it?

Reply:
? Use a single automation with a chooser, as I stated in the original PR review comment. I'm not sure what you expect from me at this point?

Reply:
the automation with a chooser like this? (Ive not tried to use this before)
```yaml
automation:
  - trigger:
      - platform: state
        entity_id: sensor.blah
    action:
      - choose:
          - conditions:
              - condition: template
                value_template: "{{ now().hour < 9 }}"
            sequence:
              - do_something```

Reply:
Yes, and instead of a template, you can use time conditions: <https://www.home-assistant.io/docs/scripts/conditions/#time-condition>

Reply:
ok - got it. I'll close this PR as its not the way you like the code, and I'll rework, then resubmit. 
I find there is a lot of info about HA, but I find the docs hard to work through. 
I’m a documentation guy, so I'll try to add things which help me make sense of it.